packages/experimental/Avatar/src/JSAvatar.types.ts

```diff
@@ -163,9 +163,9 @@ export interface JSAvatarProps extends IViewProps, AvatarConfigurableProps {
   shape?: AvatarShape;
 
   /**
-   * Currently used for image URL, will be deprecated when image prop works
+   * Shorthand for passing image URL instead of using `image` prop
    */
-  src?: string;
+  imageUrl?: string;
```

Comment:
Do you intend to remove the image prop as well?

Reply:
Not for now. Because I plan to check how Avatar is used. If we can use it without image prop than I'll remove it

Reply:
Ok, I checked usages of the Avatar. There was only one case when image was used in format `{source: {uri: 'smth.png'}}`. And I'm not sure if it was our PersonaCoin (in fabric repo), because there were other props like resolutionId. But in Office repo imageUrl were used really often

Reply:
Our win32 version of the Avatar is currently shipped as PersonaCoin/Persona. Any references to Avatar would either be for macos/ios or FluentUI web.

Our clients often have code for multiple platforms in the same repo, so I would make sure that you're looking at the right platform for that case.

Reply:
Yeah, this usage was not from win32. It was native avatar